heron/spi/src/java/com/twitter/heron/spi/common/keys.yaml

```diff
@@ -62,6 +62,12 @@ SCHEDULER_IS_SERVICE:                heron.scheduler.is.service
 SCHEDULER_JAR:                       heron.jars.scheduler
 
 ################################################################################
+# keys for Topology Update command
+################################################################################
+TOPOLOGY_UPDATE_DEACTIVATE_SECONDS:  heron.topology.update.deactivate.seconds
+TOPOLOGY_UPDATE_REACTIVATE_SECONDS:  heron.topology.update.reactivate.seconds
```

Comment:
@billonahill The activate and deactivate wait durations are used to ensure runtime states are flushed/recovered as you explained below. It may not be always feasible to predict the time in all use cases. Moreover it would be mess up the api if we add one for each use case. Would it be better if the scheduler could `callback` user implementations of some hooks to control this behavior?